Makefile

```diff
@@ -22,3 +22,12 @@ run-ci-e2e-test:
 .PHONY: clean
 clean:
 	rm -rf ${OUTPUT_DIR}
+
+.PHONY: local-run
+local-run:
+	hack/run-local.sh -a run -c "$c"
+
+.PHONY: local-clean
+local-clean:
+	hack/run-local.sh -a cleanup
```

Comment:
We should rethink if we are running the operator locally. Technically we are no longer doing that. I suggest we go with `olm-run` and `olm-clean` and name the script `olm.sh`. WDYT?

Reply:
This is obvious based on our README.